supervisor/errata_utils.py

```diff
@@ -90,8 +94,31 @@ def get_erratum(erratum_id: str | int):
         status=ErrataStatus(details["status"]),
         all_issues_release_pending=all_issues_release_pending,
         last_status_transition_timestamp=last_status_transition_timestamp,
+        comments=comments,
     )
 
+def get_erratum_comments(erratum_id: str | int) -> list[ErrataComment] | None:
+    """Get all comments for an erratum with the given erratum_id"""
+    logger.debug("Getting comments for erratum %s", erratum_id)
+    try:
+        data = ET_api_get(f"comments?filter[errata_id]={erratum_id}")
+
+        return [
+            ErrataComment(
+                id=comment_data["id"],
+                created_at=datetime.fromisoformat(
+                    comment_data["attributes"]["created_at"].replace("Z", "+00:00")
+                ),
+                errata_id=comment_data["attributes"]["errata_id"],
+                text=comment_data["attributes"]["text"],
+                who=comment_data["attributes"]["who"],
+                type=comment_data["type"],
+            )
+            for comment_data in data["data"]
+        ]
+    except Exception as e:
```

Comment:
I don't want to catch this - it would mean that if the errata tool started returning comments in a way we couldn't parse we'd process issues without passing the comments to the model. The behavior if we *don't* catch could be improved - we log the exception, do nothing, and retry the same thing in 15 minutes. (Ideally we should retry a few times and then flag the issue 'needs attention' with a comment that the process issue task is failing) but it strikes me as safer and less confusing than this.